boards/waspmote-pro/Makefile.include

```diff
@@ -1,6 +1,11 @@
 # define the cpu used by the waspmote pro board
 export CPU = atmega1281
 
+# TODO is there a bootloader on the flash ? I do not understand...
+# http://www.libelium.com/products/waspmote/hardware/
+# ARDUINO_BOOTLOADER_SIZE = 2K
+ROM_RESERVED = $(ARDUINO_BOOTLOADER_SIZE)
```

Comment:
There's a bootloader and it takes 2K, so please change.

Reply:
Any source to put as reference ?

Reply:
Well, I know it uses stk500v1 protocol, which usually is implemented in bootloaders of 2K size. I tried to find exactly the reference but no luck. Maybe someone at @Libelium might know.